docs/content/Getting Started.fsx

```diff
@@ -5,217 +5,124 @@ categoryindex: 1
 index: 1
 ---
 
-# Getting started
+# Getting started using analyzers
 
 ## Premise
 
-Analyzers that are consumed by this SDK and from Ionide are simply .NET core class libraries.  
-These class libraries expose a *value* of type [Analyzer<'TContext>](../reference/fsharp-analyzers-sdk-analyzer-1.html) which is effectively a function that has input of type [Context](../reference/fsharp-analyzers-sdk-context.html) and returns a list of [Message](../reference/fsharp-analyzers-sdk-message.html) records.
+We assume the analyzers you want to use are distributed as a nuget package.
 
-## Create project
-
-Create a new class library targeting `net6.0`
+## Using analyzers in a single project
 
+First, we need to add the [fsharp-analyzers](https://www.nuget.org/packages/fsharp-analyzers) dotnet tool to the tool-manifest.
 ```shell
-dotnet new classlib -lang F# -f net6.0 -n OptionValueAnalyzer
+dotnet tool install fsharp-analyzers
 ```
 
-Note that the assembly name needs to contain `Analyzer` in the name in order for it to be picked up.
-
-Add a reference to the analyzers SDK:
-
-```shell
-dotnet add package FSharp.Analyzers.SDK
-```
+Next, add the `PackageReference` pointing to your favorite analyzers to the `.fsproj` file of the project you want to analyzse:
 
-⚠️ Note: To utilize the analyzers in FsAutoComplete (which is subsequently utilized by Ionide), it is essential to ensure that the SDK version matches correctly.
-
-```shell
-paket add FSharp.Analyzers.SDK
+```xml
+<PackageReference Include="G-Research.FSharp.Analyzers" Version="0.1.6">
```

Comment:
Maybe add a sentence that at the time of writing only the GR analyzers are known to work with the latest cli tool. Link to https://github.com/g-research/fsharp-analyzers and https://www.nuget.org/packages/G-Research.FSharp.Analyzers. This will help if people just want to play with it.